### PR TITLE
Fix CI: Link to Pthread library explicitly

### PR DESCRIPTION
**Pull request by LeonMatthesKDAB**

### CODE REVIEW

File: tests/signal/CMakeLists.txt [hunk 1]
```diff
@@ -10,9 +10,16 @@
 
 project(test-signal VERSION 0.1 LANGUAGES CXX)
 
+
```
Comment (frankosterfeld) [inline tests/signal/CMakeLists.txt:13]: Unrelated whitespace

Comment (LeonMatthesKDAB) [reply tests/signal/CMakeLists.txt]: Fair point.

**Review by seanharmer — APPROVED** [review]
